app/src/main/java/org/schabi/newpipe/local/subscription/SubscriptionsImportFragment.java

```diff
@@ -50,11 +50,11 @@ public class SubscriptionsImportFragment extends BaseFragment {
     @State
     int currentServiceId = Constants.NO_SERVICE_ID;
 
-    private List<SubscriptionExtractor.ContentSource> supportedSources;
-    private String relatedUrl;
+    private List<SubscriptionExtractor.ContentSource> supportedSources = Collections.emptyList();
+    private String relatedUrl = null;
```

Comment:
I know this was originally set based on the function that was removed, but, if I recall correctly, a declared String in Java is already null, so this assignment is not needed.

Reply:
Probably, but this just inlines the existing code, which I don’t want to change here.

Reply:
Yeah there's no problem in making it explicit that it's initialized to null